docs/src/modules/operations/pages/projects/manage-project-access.adoc

```diff
@@ -12,7 +12,7 @@ Access to projects is controlled by assigning specific roles to users. The avail
 |View services                     |✅     |✅        |✅     |❌
 |Deploy services                   |✅     |✅        |❌     |❌
 |Update services                   |✅     |✅        |❌     |❌
-|Delete services                   |✅     |❌        |❌     |❌
+|Delete services                   |✅     |✅        |❌     |❌
```

Comment:
Was this an intentional change?

Reply:
Think it got picked up in the rebase to java-spi...